CONTRIBUTING.rst

```diff
@@ -107,15 +107,115 @@ category. You can quickly list these by searching the Red Hat bugzilla for bugs
 
 Patches for bugs without keywords are welcome, too!
 
+WebUI development
+-----------------
+
+For good starting points how to develop and test Web UI interface of Anaconda see ``ui/webui/README.rst`` and ``ui/webui/test/README.rst``.
+
+PatternFly
```

Comment:
I would not introduce this section here, as it's webui only related. if you want you can add a note about patternfly in the ui/webui/README.rst